packages/react-devtools-shell/src/app/index.js

```diff
@@ -12,7 +12,7 @@ import Iframe from './Iframe';
 import EditableProps from './EditableProps';
 import ElementTypes from './ElementTypes';
 import Hydration from './Hydration';
-import InlineWarnings from './InlineWarnings';
+// import InlineWarnings from './InlineWarnings';
```

Comment:
🤪

Reply:
This change was also reverted by a later commit.

Reply:
Oh weird. You're right. I wonder why I didn't see them yesterday 😢 Sorry Luna. I'll fix.